pkg/etw/fieldopt.go

```diff
@@ -66,6 +74,17 @@ func StringArray(name string, values []string) FieldOpt {
 	}
 }
 
+// UnicodeStringArray adds an array of UTF-16 strings to the event.
+func UnicodeStringArray(name string, values []string) FieldOpt {
+	return func(em *eventMetadata, ed *eventData) {
+		em.writeArray(name, inTypeUnicodeString, outTypeString, 0)
```

Comment:
Do we want the out type here? It looks like `TraceLoggingWideString` in `TraceLoggingProvider.h` just uses `TlgInUNICODESTRING` with no out type.

Reply:
Please see below.